src/plot_api/to_image.js

```diff
@@ -61,11 +61,12 @@ function toImage(gd, opts) {
                 setTimeout(function() {
                     var svg = Snapshot.toSVG(clonedGd);
 
-                    var canvasContainer = window.document.createElement('div');
-                    var canvas = window.document.createElement('canvas');
+                    var canvasContainer = document.createElement('div'),
+                        canvas = document.createElement('canvas');
 
-                    canvasContainer.appendChild(canvas);
+                    // no need to attach canvas container to DOM
```

Comment:
Probably can ⚡ the comment. 

Also, _do we even the canvasContainer_? Can't we just use the `canvas` element directly? Not 🚫  but it's always nice to remove code.